Dockerfile

```diff
@@ -44,6 +44,6 @@ ENV HOME /opt/wildfly
 WORKDIR /opt/wildfly/source
 USER    wildfly
 
-EXPOSE 7600 8080 8787 9990 9999
+EXPOSE 8080
```

Comment:
1) you can still connect to any port you want on a container, even if it doesn't "expose" it.  they're just hints.
2) i'm not certain, but hopefully our system does the ordering numerically, not by whatever order you put them in the dockerfile, because that would be incredibly non-deterministic.